cspell.yaml

```diff
@@ -234,6 +234,13 @@ words:
   - xplat
   - xxsubtype
   - yamls
+  - arraya
+  - primitivea
+  - recorda
+  - Hdvcmxk
+  - ints
+  - myname
+  - azuresdkcadlranch
```

Comment:
think that name shouldn't be in there wherever it is

Reply:
This is the name for storage account. So, I am keeping it

Reply:
I don't think we should use the same one nor setup the coverage upload yet. We'll need some slightly different structure

Reply:
Done. Deleted `azuresdkcadlranch` and removed the scripts `upload-manifest` and `upload-coverage` in the `package.json` for now